plugin/src/main/java/com/denizenscript/denizen/objects/CuboidTag.java

```diff
@@ -957,9 +957,9 @@ public static void register() {
         // @mechanism CuboidTag.add_member
         // @description
         // Returns a modified copy of this cuboid, with the input cuboid(s) added at the end.
+        // You can also use <@link mechanism add_member>
         // @example
         // # Creates a new cuboid named "my_third_cuboid" and adds "my_cuboid" and "my_second_cuboid" as members.
-        // # You can also use the "add_member" mechanism.
```

Comment:
The mech itself is already linked, these notes are intentionally separately attached to the examples